dynamic_programming/minimum_steps_to_one.py

```diff
@@ -1,7 +1,7 @@
 """
 YouTube Explanation: https://www.youtube.com/watch?v=f2xi3c1S95M
 
-Given an integer n, return the minimum steps to 1
+Integer n is given, return the minimum steps from n to 1
```

Comment:
```suggestion
Given an integer n, return the minimum steps from n to 1
```
The phrasing in the first part of the sentence was more natural